tests/check/tests.yaml

```diff
@@ -0,0 +1,1424 @@
+          - object: document:3
+            relation: writer
+            user: user:cheetah
+        assertions:
+          - tuple:
+              object: document:1
+              relation: viewer
+              user: user:aardvark
+            expectation: false
+          - tuple:
+              object: document:2
+              relation: viewer
+              user: user:badger
+            expectation: false
+          - tuple:
+              object: document:3
+              relation: viewer
+              user: user:cheetah
+            expectation: true
+
+  - name: ExclusionAndExclusionInSubtract
+    stages:
+      - model: |
+          type user
+          
+          type document
+            relations
+              define writer: [user] as self
+              define editor: [user] as self
+              define owner: [user] as self
+              define viewer as writer but not (editor but not owner)
+        tuples:
+          - object: document:1
+            relation: writer
+            user: user:aardvark
+          - object: document:1
+            relation: editor
+            user: user:aardvark
+          - object: document:1
+            relation: owner
+            user: user:aardvark
+          - object: document:2
+            relation: writer
+            user: user:badger
+          - object: document:2
+            relation: editor
+            user: user:badger
+          - object: document:3
+            relation: writer
+            user: user:cheetah
+          - object: document:3
+            relation: owner
+            user: user:cheetah
+        assertions:
+          - tuple:
+              object: document:1
+              relation: viewer
+              user: user:aardvark
+            expectation: true
+          - tuple:
+              object: document:2
+              relation: viewer
+              user: user:badger
+            expectation: false
+          - tuple:
+              object: document:3
+              relation: viewer
+              user: user:cheetah
+            expectation: true
+
+  - name: userset_as_user
+    stages:
+      - model: |
+          type user
+          
+          type group
+            relations
+              define member: [user] as self
+    
+          type document
+            relations
+              define viewer: [group#member] as self
+        tuples:
+          - object: document:1
+            relation: viewer
+            user: group:x#member
+        assertions:
+          - tuple:
+              object: document:1
+              relation: viewer
+              user: group:x#member
+            expectation: true
+
+  - name: wildcard_direct
+    stages:
+      - model: |
+          type user
+          
+          type document
+            relations
+              define viewer: [user:*] as self
+        tuples:
+          - object: document:public
+            relation: viewer
+            user: user:*
+        assertions:
```

Comment:
Mind adding one more assertion for the direct check on the `user:*` tuple as well?

```
- tuple:
     object: document:public
     relation: viewer
     user: user:*
  expectation: true
```

Reply:
Yep! Done. Thanks.